core/src/main/scala/cats/data/Xor.scala

```diff
@@ -166,9 +166,19 @@ private[data] sealed abstract class XorInstances extends XorInstances1 {
       def combine(x: A Xor B, y: A Xor B): A Xor B = x combine y
     }
 
-  implicit def xorBifunctor: Bifunctor[Xor] =
-    new Bifunctor[Xor] {
+  implicit def xorBifunctor: Bifunctor[Xor] with Bifoldable[Xor] =
```

Comment:
It looks like we are combining the `Bifunctor` and `Bifoldable` instances in some places but not others. I guess if we are going to go forward with #800 then they will eventually be combined anyway. Was there any particular reason behind when they've been combined and when they haven't?